helm-charts/charts/warehouse/values.yaml

```diff
@@ -0,0 +1,198 @@
+# Ensure that the warehouse instance name differs from the starrocks instance name to avoid conflicts. For example, both
+# will generate configmaps using the 'nameOverride' as a prefix to the configmap name, followed by a specific suffix.
+nameOverride: "kube-starrocks"
+
+# TimeZone is used to set the environment variable TZ for pod, with Asia/Shanghai as the default.
+timeZone: Asia/Shanghai
+
+# This configuration is used to integrate with external system DataDog.
+# You can enable the integration by setting the enabled to true, e.g. datalog.log.enabled=true will enable datadog agent
+# to collect the log.
+datadog:
+  log:
+    enabled: false
+    # besides the attributes you added, chart will append "source" and "service" attributes to the log config.
+    # see https://docs.datadoghq.com/containers/kubernetes/log/?tab=operator for more details.
+    logConfig: '{}' # e.g. '{"app": "starrocks", "tags": ["aa", "bb"]}'
+  metrics:
+    enabled: false
+
+spec:
+  # The name of starrocks cluster.
+  # Note: the starrocks cluster must be created before the warehouse by operator.
+  starRocksClusterName:
```

Comment:
may need to specify this field must not be empty?

Reply:
done